server/authentication.js

```diff
@@ -1,15 +1,14 @@
 const expressjwt = require('express-jwt')
 const jwksRsa = require('jwks-rsa')
 const axios = require('axios')
-const consola = require('consola')
 const { findUserByEmail } = require('./controllers/users.js')
 
 const getToken = req => {
   const tokenString = req.cookies['auth._token.auth0']
     ? req.cookies['auth._token.auth0']
     : req.headers.authorization
   const tokenMatch = /(?<=Bearer ).+/.exec(tokenString)
-  return tokenMatch.length > 0 ? tokenMatch[0] : null
+  return (tokenMatch || []).length > 0 ? tokenMatch[0] : null
```

Comment:
Not sure if optional chaining is in this branch, but is a good spot to use it

Reply:
Unfortunately they aren't :'(